.pre-commit-config.yaml

```
@@ -1,4 +1,23 @@
 repos:
+  - repo: local
+    hooks:
+      - id: avoid-old-website-urls
+        name: Avoid old URLs (https://elastisys.io/compliantkubernetes)
+        entry: "elastisys.io/compliantkubernetes"
+        language: pygrep
+        files: ".*[.]md$"
+      - id: avoid-old-names
+        name: Use Welkin consistently
```

```suggestion
        name: Avoid old names and use use "Welkin" instead
```
To be consistent with the other one.